scripts/transfer_approve_deposit.js

```diff
@@ -18,8 +20,8 @@ module.exports = async (callback) => {
     const GnosisSafe = artifacts.require("GnosisSafe")
     const masterSafe = await GnosisSafe.at(argv.masterSafe)
 
-    const deposits = require(argv.depositFile)
-    // TODO - make a simpler to construct deposit file style
+    const deposits = JSON.parse(await fs.readFile(argv.depositFile, "utf8"))
```

Comment:
I am struggling to understand why this change is necessary.

Reply:
The difference is that `require` uses the path of the script as the base path, while `fs.readFile` uses the current working directory.
So, if your current working directory is `/path/to/dex-liquidity-provision/` and your deposit file is in `./examples/deposits.json`, then before this PR you'd have to run:
```
npx truffle exec scripts/transfer_approve_deposits.js [...] --depositFile="../examples/deposits.json"
```
(Note the two dots in the relative path.)
Now you can just use:

``` 
npx truffle exec scripts/transfer_approve_deposits.js [...] --depositFile="./examples/deposits.json"
```
(Note the single dots in the relative path.)
I noticed this bug when I tried to rely on the autocomplete to fill the path, which doesn't work because it autocompleted based on the current working directory.

Reply:
Thanks for clearing that up! I now understand.